impeller/entity/contents/clip_contents.cc

```diff
@@ -86,7 +86,6 @@ bool ClipContents::Render(const ContentContext& renderer,
     {
       cmd.label = "Difference Clip (Increment)";
 
-      cmd.primitive_type = PrimitiveType::kTriangleStrip;
```

Comment:
How did we manage to get rid of the strip here without reworking the vertex buffer? Was it just incorrect?

Reply:
@chinmaygarde it was always overwritten, see line: 123. I think there was a refactor at some point to use the `geometry_result` but this line never got removed.

Reply:
Got it.